docs/config.md

```diff
@@ -212,6 +212,9 @@ bls_composing_models: <comma-delimited-string-list>
 # Maximum request rate used for the automatic/quick config search
 [ run_config_search_max_request_rate: <int> | default: 8092 ]
 
+# Maximum number of steps taken during binary concurrency search
```

Comment:
Is there any chance we add support for binary request rate? Should this be a more generic 'binary' option so we don't end up with `max_brs_steps` as well?  `run_config_search_max_binary_search_steps` (?)

Reply:
Do we also want something (a magic -1 value?) to mean "Do a full binary search until you find the exact boundary"?

Reply:
I don't think that's necessary

Reply:
So the decision is to keep concurrency (or 'c') in the option name?

Reply:
To the first question, that's a good point. I'll make the name and description more generic